tests/network-tests/src/constantinople/tests/impl/electingCouncil.ts

```diff
@@ -17,18 +17,19 @@ export function councilTest(m1KeyPairs: KeyringPair[], m2KeyPairs: KeyringPair[]
   const K: number = +process.env.COUNCIL_ELECTION_K!;
   const greaterStake: BN = new BN(+process.env.COUNCIL_STAKE_GREATER_AMOUNT!);
   const lesserStake: BN = new BN(+process.env.COUNCIL_STAKE_LESSER_AMOUNT!);
-  const defaultTimeout: number = 120000;
+  const defaultTimeout: number = 300000;
```

Comment:
I don't understand why the timeout had to be increased here, and in some other tests also.

Wouldn't it be better if the timeout value for each scenario was derived based on how many finalized blocks the tests should be completed within, and the chain's block interval?

Its also not clear, is this timeout for the entire suite or per `.test()`? If it is per test then the timeout should be specific to each test, not the max value to accommodate all the tests.

Reply:
The timeout time now is defined using number of blocks required for the test to pass.